scripts/check_calendars.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+import os
+from icalendar import Calendar
+
+def get_ics(filename):
+    return filename.endswith('ics')
+
+def check_if_correct_parse(ics_file):
+    fh = open(ics_file, 'rb')
+    try:
+        # some calendars, such as Austrian ones have multiple
+        # vCalendar entries - we probably don't want them to fail
+        # parse. So we set multiple=True below
+        cal_entries = Calendar.from_ical(fh.read(), multiple=True)
+        if cal_entries is None:
+            raise ValueError
+    except ValueError as ve:
+        print ve.message
+        raise
+    finally:
+        fh.close()
+
```

Comment:
2 blank lines.

also it would be best (though not required) to nest all of this uner a `if __name__ == '__main__':`